### PR TITLE
Merge 9.0.x up into 10.0.x 

### PR DESCRIPTION
**Pull request by simPod**
@greg0ire How to process this properly? I'd need https://github.com/doctrine/coding-standard/pull/269 to be include in 10.0.x branch.

### CODE REVIEW

**Comment by greg0ire** [comment]
What you did is correct, but you need to fix the build now.

**Review by ostrolucky — DISMISSED** [review]

**Review by malarzm — DISMISSED** [review]

**Comment by Ocramius** [comment]
@simPod can you adjust the PR title to match meaning? For release notes

**Comment by simPod** [comment]
@Ocramius what about now

**Comment by Ocramius** [comment]
Ok, then it seems wrong to do this  :D

**Comment by simPod** [comment]
lol I messed up. Now it's correct.

**Review by Ocramius — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -28,9 +28,16 @@ update-compatibility-patch-80:
...
 	@until [ "$${compatibility_resolved}" == "y" ]; do read -p "Have finished your changes (y|n)? " compatibility_resolved; done && compatibility_resolved=
-	@git diff -- tests/expected_report.txt tests/fixed tests/input > .tmp-patch-80 && mv .tmp-patch-80 tests/php80-compatibility.patch && git apply -R tests/php80-compatibility.patch
+	@git diff -- tests/expected_report.txt tests/fixed tests/input > .tmp-patch && mv .tmp-patch tests/php80-compatibility.patch && git apply -R tests/php80-compatibility.patch
 	@git commit -m 'Update compatibility patch' tests/php80-compatibility.patch
 
+update-compatibility-patch-81:
+	@git apply tests/php81-compatibility.patch
+	@printf "Please open your editor and apply your changes\n"
+	@until [ "$${compatibility_resolved}" == "y" ]; do read -p "Have finished your changes (y|n)? " compatibility_resolved; done && compatibility_resolved=
+	@git diff -- tests/expected_report.txt tests/fixed tests/input > .tmp-patch && mv .tmp-patch tests/php81-compatibility.patch && git apply -R tests/php81-compatibility.patch
```
Comment (simPod) [inline Makefile:38]: I had to add `input` folder here, I accidentally threw away php81 tests

**Review by Ocramius — APPROVED** [review]

**Review by ostrolucky — APPROVED** [review]

**Review by malarzm — APPROVED** [review]

**Comment by malarzm** [comment]
Thanks @simPod!

**Pull request merged**